cms/static/js/views/video_thumbnail.js

```diff
@@ -191,7 +239,11 @@ define(
                 } else if (this.action === 'edit') {
                     this.setActionInfo(this.action, true);
                 }
-                this.$('.thumbnail-wrapper').addClass('focused');
+
+                // When we had error, focused effect was not wearing off after hover out.
+                if (!$(this.$el.parent()).hasClass('has-thumbnail-error')) {
+                    this.$('.thumbnail-wrapper').addClass('focused');
+                }
```

Comment:
we need this because When we had error, focused effect was not wearing off even after hover out on the image containing error. `this.$('.thumbnail-wrapper').addClass('focused'); ` adds to all elements, however, here, we want to skip those with error so as to avoid focussed effect issue.